charts/newrelic-logging/templates/_host_network.tpl

```diff
@@ -0,0 +1,20 @@
+{{/* Returns whether the we should run with hostNetwork: true based on the user configuration. */}}
+{{- define "newrelic-logging.hostNetwork" -}}
```

Comment:
Hi! I can not see any usage of these two definitions. Do we really need this file at all?

Reply:
@luckslovez if the tests pass without it, it can be removed.
this file was just copied from other NR helm charts and updated accordingly 
I will remove it